ntbs-service/Models/NotificationSite.cs

```diff
@@ -11,6 +12,7 @@ public class NotificationSite
         public int SiteId { get; set; }
         public Site Site { get; set; }
 
+        [RequiredIf("SiteId == SiteId.OTHER && Notification.ShouldValidateFull", ErrorMessage = ValidationMessages.DiseaseSiteOtherIsRequired)]

```

Comment:
I think this should just happen regardless of `Notification.ShouldValidateFull`. Let's change it to 
`[RequiredIf("SiteId == SiteId.OTHER", ErrorMessage = ValidationMessages.DiseaseSiteOtherIsRequired)]`?

Reply:
https://airelogic-nis.atlassian.net/wiki/spaces/R2/pages/146538765/Notification+Validation

Validation page shows that it's required on Full Validation only, we can discuss it with Nancy

Reply:
You're right there. Seems odd, but let's leave as is for the time being then :)